pkgs/swift2objc/lib/src/config.dart

```diff
@@ -32,12 +34,21 @@ class Config {
   /// intermediate files after generating the wrapper
   final Uri? tempDir;
 
-  const Config({
-    required this.input,
-    required this.outputFile,
-    this.tempDir,
-    this.preamble,
-  });
+  /// Filter function to filter APIs
+  ///
+  /// APIs can be filtered by name
+  ///
+  /// Includes all declarations by default
+  final bool Function(Declaration declaration)? include;
```

Comment:
This should be non-nullable. `final bool Function(Declaration declaration) include;`